### DISCUSSION TITLE
Chromium SharedWorker fails to load importScripts from worker.js

### DISCUSSION THREAD

**Discussion by rockerBOO** [wasm-bindgen/wasm-bindgen#3733] [Q&A]
### Describe the Bug

When using SharedWorker, the worker importScripts fails to load due to DOM exception in Chromium Version 119.0.6045.159 (Official Build, ungoogled-chromium) Arch Linux (64-bit).

### Steps to Reproduce

I took the web workers code from https://rustwasm.github.io/wasm-bindgen/examples/wasm-in-web-worker.html and exchanged in SharedWorker and added the SharedWorker and MessagePort feature. 

```
git clone https://github.com/rockerBOO/shared-worker-debug-test
wasm-pack build --target no-modules
python -m http-server 
```
Then in Chromium the address http://localhost:8000, and in a separate tab go to chrome://inspect/#workers and inspect the shared worker.

### Expected Behavior

Exchanging out the Worker to SharedWorker should work in Chromium. Works in Chrome,Firefox though. 

### Actual Behavior

DOMException when trying to load the web workers script.

```
Uncaught DOMException: Failed to execute 'importScripts' on 'WorkerGlobalScope': The script at 'http://localhost:8081/pkg/wasm_in_web_worker.js' failed to load.
    at http://localhost:8081/worker.js:5:1
(anonymous) @ worker.js:5
```

### Additional Context

I'm realizing this now as it only seems to happen in chromium. Thought it was initially as Chrome related. Did a test in Chrome and it works as expected though.

**Comment by LucaCappelletti94** [comment]
Hi! I am encountering the same issue, not only on Chromium. See more here [on this other GitHub issue](https://github.com/trunk-rs/trunk/issues/778). Did you find any solution?

**Comment by daxpedda** [comment]
I think the error message is quite correct, not all browsers actually support `importScripts` in every context yet, though I don't know what the current state is.
